packages/nimbus/src/components/calendar/calendar.dev.mdx

```diff
@@ -0,0 +1,502 @@
+
+All date values **must** use `@internationalized/date` types:
+- `CalendarDate` for date-only values (recommended for most use cases)
+- `CalendarDateTime` for date with time
+- `ZonedDateTime` for date with time and timezone
+
+### Accessibility
+
+The Calendar handles most accessibility requirements internally. However, you must always associate an internationalized label with the component. Visual labels are preferable, and can be set by:
+
+- Associating a `<label>` element with the Calendar using `aria-labelledby`:
+
+```tsx
+<label id="calendar-label">
+  {intl.formatMessage({ id: 'selectDate', defaultMessage: 'Select date' })}
+</label>
+<Calendar aria-labelledby="calendar-label" />
+```
+
+- Associating a `<label>` element with the Calendar using `htmlFor`:
+
+```tsx
+<label htmlFor="calendar-id">
+  {intl.formatMessage({ id: 'selectDate', defaultMessage: 'Select date' })}
+</label>
+<Calendar id="calendar-id" />
+```
+
+If your design requires that the label should not be visible, the label should be set using the `aria-label` prop:
+
+```tsx
+<Calendar aria-label={intl.formatMessage({ id: 'selectDate', defaultMessage: 'Select date' })} />
+```
+
+If your use case requires tracking and analytics for this component, it is good practice to add a **persistent**, **unique** id to the component:
+
+```tsx
+const PERSISTENT_ID = "booking-calendar";
+
+export const BookingForm = () => (
+  <Calendar id={PERSISTENT_ID} />
+);
+```
+
+#### Keyboard navigation
+
+The component supports full keyboard interaction:
+- `Tab` / `Shift+Tab`: Move focus into and out of the calendar
+- `Arrow keys`: Navigate between dates within the grid
+- `Page Up` / `Page Down`: Navigate to the previous/next month
+- `Shift + Page Up` / `Shift + Page Down`: Navigate to the previous/next year
+- `Home` / `End`: Navigate to the start/end of the current week
+- `Enter` / `Space`: Select the focused date
+
+## API reference
+
+<PropsTable id="Calendar" />
+
+## Common patterns
+
+### Booking system with constraints
+
+A typical booking system that prevents past dates and weekends:
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState<CalendarProps<DateValue>["value"]>(null);
+  const today = new CalendarDate(2025, 1, 15);
+
+  return (
+    <Stack direction="column" gap="400">
+      <Text fontWeight="600">Select your booking date</Text>
+      <Text fontSize="sm" color="neutral.11">
+        Available dates: Weekdays only, starting tomorrow
+      </Text>
+      <Calendar
+        value={value}
+        onChange={setValue}
+        minValue={today.add({ days: 1 })}
+        isDateUnavailable={(date) => {
+          const dayOfWeek = date.toDate('UTC').getDay();
+          return dayOfWeek === 0 || dayOfWeek === 6;
+        }}
+      />
+      {value && (
+        <Text fontSize="sm">
+          Selected booking date: {value.toString()}
+        </Text>
+      )}
+    </Stack>
+  );
+}
+```
+
+### Working days selection
```

Comment:
Similar situation to above. 
Before & after glamour shots (selected & display dates off).

<img width="400" height="557" alt="Image" src="https://github.com/user-attachments/assets/6093c0d1-dfac-4579-8cab-1ec666d7790f" />

<img width="400" height="558" alt="Image" src="https://github.com/user-attachments/assets/8511b559-766e-4db5-89fd-bab2d86f9545" />